data/jibauni/rebel jibauni missions.txt

```diff
@@ -0,0 +1,119 @@
+					decline
+			`	The Jibauni is coming faster now. When it gets within speaking distance you hear it ask, "Can anyone take me to <planet>? I need to leave immediately."`
+			choice
+				`	(Offer to transport the Jibauni.)`
+				`	(Ignore the Jibauni.)`
+					decline
+			`	You ask the Jibauni if you would fit the bill for transportation. It immediately thanks you and pulls you towards the <ship>, seemingly wanting to go.`
+			choice
+				`	(Follow the Jibauni and leave with it.)`
+					goto followandleave
+				`	"Whoa, not so fast there."`
+			`	"We've got to go now," the Jibauni exclaims. "Come on!" It pulls you into your ship and forces you to launch.`
+				launch
+			label followandleave
+			`	You take off upon reaching your ship. Rebel Jibauni ships launch with you.`
+				launch
+	on enter
+		"old reputation: Rebel Jibauni" = "reputation: Jibauni (Rebel)"
+		"reputation: Jibauni (Rebel)" <?= -1
+		conversation
+			`The Jibauni seems somewhat calmer now. "My name is Qui-ni," it says. "You are taking me to <destination>, right?"`
+			choice
+				`	"Well, I did leave with you."`
+				`	"Not willingly."`
+					goto notwilling
+			`	"Good. Now, please evade the Rebel Jibauni who are chasing us. Let's go immediately."`
+				goto end
+			label notwilling
+			`	"Really? Well, you can take me to <destination> immediately and be rid of me."`
+			label end
+	on complete
+		"reputation: Jibauni (Rebel)" = "old reputation: Rebel Jibauni"
+		clear "old reputation: Rebel Jibauni"
+		payment 500
+		conversation
+			`When you finally settle into the allotted dock, Qui-ni pays you <payment> with a relieved look. "I'm glad to be back," it says. Qui-ni then walks past you and out the door of the <ship>, leaving you to wonder what in space that was about.`
+	npc
+		government "Jibauni (Rebel)"
+		personality entering staying
+		fleet "Small Rebel Jibauni Military"
+	npc
+		government "Jibauni (Rebel)"
+		personality launching waiting
+		fleet "Small Rebel Jibauni Military" 2
+
+
+mission "Rebel Jibauni: Scam Government Transportation 1"
+	landing
+	name "Take a Rebel to <planet>"
+	description "Take a Rebel Jibauni to <destination>."
+	source
+		government "Jibauni (Rebel)"
+	destination
+		government "Jibauni (Rebel)"
+		distance 1 2
+	passengers 1
+	to offer
+		has "event: rebel jibauni: initial transportation"
+		has "Rebel Jibauni: Landed on Andwerfen: declined"
+		has "rebel jibauni: will transport upon request"
+		random < 80
+	on offer
+		conversation
+			`A Rebel arrives panting at the door of the <ship> as it opens. "Take me to <destination>," they say. "This has been requested by Rebel Jibauni."`
+			choice
+				`	"Alright."`
+					accept
+				`	"Isn't that the wrong wording?"`
+			`	"W-wrong wording? Wha-what do you mean?" queries the Jibauni. A commotion has arisen in the spaceport behind them.`
+			choice
+				`	"Never mind. Come aboard."`
+					accept
+				`	"Others have told me this with the words, 'This has been Requested by the Real Jibauni'."`
```

Comment:
Elsewhere the key phrasing is solely 'Requested by the Real Jibauni'
```suggestion
				`	"Others have told me this with the words that this has been 'Requested by the Real Jibauni'."`
```
Or if you are going for that more extensive phrasing then "requested" shouldn't be capitalized as well
```suggestion
				`	"Others have told me this with the words, 'This has been requested by the Real Jibauni'."`
```